src/fluent/watch.ts

```diff
@@ -29,24 +30,28 @@ export enum WatchEvent {
   DATA = "data",
   /** Bookmark is received */
   BOOKMARK = "bookmark",
-  /** ResourceVersion is updated */
+  /** @deprecated */
   RESOURCE_VERSION = "resource_version",
   /** 410 (old resource version) occurs */
   OLD_RESOURCE_VERSION = "old_resource_version",
   /** A reconnect is already pending */
   RECONNECT_PENDING = "reconnect_pending",
+  /** Resource list operation run */
+  LIST = "list",
+  /** List operation error */
+  LIST_ERROR = "list_error",
 }
 
 /** Configuration for the watch function. */
 export type WatchCfg = {
   /** Whether to use bookmarks with the watch. */
   allowWatchBookmarks?: boolean;
-  /** The resource version to start the watch at, this will be updated on each event. */
-  resourceVersion?: string;
```

Comment:
Breaking change not that it is a big deal, not sure that anyone was using this

Reply:
Correct these are breaking, will need to update the pr to note the breaking changes